erts/preloaded/src/erlang.erl

```diff
@@ -7475,6 +7475,11 @@ reported to the owning process using signals of the form
 
 The maximum number of ports that can be open at the same time can be configured
 by passing command-line flag [`+Q`](erl_cmd.md#max_ports) to [erl](erl_cmd.md).
+
+When the VM shuts down, spawned executables are sent `SIGTERM` on unix. The
```

Comment:
"unix" -> "POSIX"?  Hopefully this can be reimplemented for win32 in later work, anyway.  Is it okay to have divergent behavior on the platforms?